src/Symfony/Component/Messenger/Bridge/Amqp/Transport/Connection.php

```diff
@@ -60,9 +60,18 @@ class Connection
     ];
 
     private const AVAILABLE_QUEUE_OPTIONS = [
+        'bindings',
+        'flags',
+        'arguments',
+    ];
+
+    private const ORIGINAL_BINDING_KEYS = [
         'binding_keys',
         'binding_arguments',
-        'flags',
```

Comment:
Will all DSN strings that work in 5.1 still work after merging this PR?

Reply:
Yes, this is fully backwards compatible. Only deprecation notices are introduced for the binding_keys and binding_arguments.